core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+    def read_argument(self) -> dict | None:
+        arg_type = self.read_u8()
+        if arg_type is None:
+            return None
+
+        if arg_type == ARG_TYPE_INPUT:
+            data = self.read_bytes(2)
+            if data is None:
+                return None
+            index = int.from_bytes(data, "little")
+            return {"type": "Input", "index": index}
+        elif arg_type == ARG_TYPE_GAS_COIN:
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)